internal/controller/serving/inferenceservice_controller_test.go

```diff
@@ -766,6 +777,29 @@ var _ = Describe("InferenceService Controller", func() {
 				_ = createServingRuntime(testNs, UnsupprtedMetricsServingRuntimePath)
 				SklearnInferenceService := createInferenceService(testNs, UnsupportedMetricsInferenceServiceName, UnsupportedMetricsInferenceServicePath)
 
+				Expect(k8sClient.Delete(ctx, SklearnInferenceService)).Should(Succeed())
+				Eventually(func() error {
+					configmap := &corev1.ConfigMap{}
+					key := types.NamespacedName{Name: UnsupportedMetricsInferenceServiceName + constants.KserveMetricsConfigMapNameSuffix, Namespace: SklearnInferenceService.Namespace}
+					err := k8sClient.Get(ctx, key, configmap)
+					return err
+				}, timeout, interval).ShouldNot(Succeed())
```

Comment:
It checks if the configmap is deleted or not so I think the timeout could be set less than default value(20s). serving unit test took around 200~350s so I want to reduce the time changing this with 10s for timeout, and 4s for interval.